suzieq/cli/sqcmds/ArpndCmd.py

```diff
@@ -39,8 +39,12 @@ def __init__(
               description="IP address, in quotes, to qualify output")
     @argument("macaddr",
               description="MAC address, in quotes, to qualify output")
+    @argument("prefix",
```

Comment:
The text string maybe too long to display well. Have you checked? Alternately, a shorter description might be "Show all addresses in this subnet prefix"